Library/Homebrew/cli/parser.rb

```diff
@@ -282,17 +294,23 @@ def name_to_option(name)
         end
       end
 
+      sig { params(names: String).returns(T.nilable(String)) }
       def option_to_description(*names)
         names.map { |name| name.to_s.sub(/\A--?/, "").tr("-", " ") }.max
```

Comment:
i think this logic is wrong (pretty sure it's choosing alphabetically, which seems off), but i left it alone

Reply:
I agree. It's only used in the `#option_description` method as a fallback if an explicit description doesn't exist. It'd probably make more sense to grab the max by string length instead. I think all of our internal commands have descriptions so it doesn't really matter much for us. I'm fine with fixing it here or as a follow-up though.